### PR TITLE
Updated index file

### PR DESCRIPTION
**Pull request by chandresh1339**
Dialog pop up confirming that the user wants to delete the biome.

### CODE REVIEW

File: app/Http/Controllers/ChargeController.php [hunk 1]
```diff
@@ -38,6 +38,16 @@ public function json()
...
 
+    <!--Add ability to search charge by tag, not just name #18->
+        
+     public function searchByTag(Request $request)
+        {
+        $tags = $request->input('tag');
+
+        $charges = Charge::where('tag', 'like', "%$tag%")->orderBy('tag')->paginate(15);
+//This will check the serch methos in folder 'charge' under file 'charge.searchByTag'.
+        return view('charge.searchByTag', ['charges' => $charges]);
```
Comment (BenOvermyer) [inline app/Http/Controllers/ChargeController.php]: This view doesn't resolve.

File: resources/views/resource/index.blade.php [hunk 2]
```diff
@@ -20,7 +20,37 @@
             <p class="flex">
                 <a href="{{ route('resource.create') }}" class="btn">Create New</a>
                 <a href="{{ route('resource.json') }}" class="btn">Create from JSON</a>
+                <!--UI to show the Delete button-->
```
Comment (BenOvermyer) [inline resources/views/resource/index.blade.php:23]: None of the changes to this file are appropriate for this PR.

**Review by BenOvermyer — CHANGES_REQUESTED** [review]
"Updated index file" doesn't tell me what this pull request is meant to do.

From the commits, it looks like it's implementation of tag searching for charges. However, it also includes broken changes to the resource index view. Please remove that file's changes from this PR, and address the other comments.

File: resources/views/resource/index.blade.php (hunk 2 above)
Comment (chandresh1339) [reply resources/views/resource/index.blade.php:23]: PR removed. I apologize for it.

**Pull request closed without merging**